.pre-commit-config.yaml

```diff
@@ -1,5 +1,37 @@
 repos:
--   repo: https://github.com/ambv/black
-    rev: 18.9b0
+  - repo: https://github.com/python/black
+    rev: stable
     hooks:
-    - id: black
+      - id: black
+  - repo: https://github.com/codespell-project/codespell
+    rev: v1.16.0
+    hooks:
+      - id: codespell
+        args:
+          - --quiet-level=2
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.9
+    hooks:
+      - id: flake8
+        args:
+          - --max-complexity=10
+          - --max-line-length=88
+  - repo: https://github.com/timothycrosley/isort
```

Comment:
I prefer https://github.com/asottile/reorder_python_imports rather than isort as the former reorders everything statically.